.github/workflows/test.yml

```diff
@@ -58,3 +58,31 @@ jobs:
       with:
         path: /home/runner/.artefacts
         name: artefacts
+
+  test-macos:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest]
+        go_version: ['1.14.x']
+        vim_version: ["master"]
+    runs-on: ${{ matrix.os }}
+    env:
+      VIM_FLAVOR: vim
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@722adc63f1aa60a57ec37892e133b1d319cae598
+    - name: Install Vim
+      uses: ./github/actions/setupvim
```

Comment:
Is the idea to keep the action as a part of the `govim/govim` repo?
Reason for asking is that the `README.md` header points to another (non-existing/private) repo, and the `go.mod` have a module name that is close to real but doesn't exist either.

Reply:
> Is the idea to keep the action as a part of the govim/govim repo?

For now, yes. As you have keenly observed, it originated elsewhere as I was hacking around writing https://blog.myitcv.io/2020/02/04/portable-ci-cd-with-pure-go-github-actions.html. Doesn't really matter where it lives to be honest, because I don't suspect it will live for any reason other than our own use.